js/vnext/projection/cells.js

```diff
@@ -31,21 +31,18 @@ function translateRow(columnGroup, row, group) {
     const obj =  {
       classes: row.classes,
       cells: _.map(columnGroup.leafColumns, col => {
-        const cell = row.item[col.name] || {};
-        const classes = col.classes;
-        const template = col[group + 'Template'];
-        let html = '';
-        if (cell.html) {
-          html = cell.html;
+        const cell = { classes: col.classes, attributes: {} };
+        if (col.template) {
+          cell.html = col.template(row.item);
+        } else if (col.value) {
+          cell.value = col.value(row.item);
+        } else if (col.field) {
+          cell.value = _.chain(col.field.split('/')).reduce((memo, key) => (memo[key]), row.item).value();
```

Comment:
Almost there, just 1 problem, what if `row.item[key]` is `undefined`? You'll get exception for the next iteration.
```js
cell.value = _.chain(col.field).split(/\.\//).reduce((memo, key) => (memo || {})[key], row.item).value();
```